src/include/daos_srv/pool.h

```diff
@@ -27,7 +27,7 @@
 /*
  * Aggregation of pool/container/object/keys disk format change.
  */
-#define DS_POOL_GLOBAL_VERSION		1
+#define DS_POOL_GLOBAL_VERSION		2
```

Comment:
Trying to remember all of the aspects of interop management: does DS_POOL_GLOBAL_VERSION always need to be kept at the same value as DAOS_POOL_GLOBAL_VERSION defined in include/daos/pool.h?

Reply:
DS_POOL_GLOBAL_VERSION is not used any more, we now use DAOS_POOL_GLOBAL_VERSION, we might need remove it for clarity.